src/account/interface.ts

```diff
@@ -39,9 +40,10 @@ export abstract class AccountInterface extends ProviderInterface {
    *
    * @returns response from addTransaction
    */
-  public abstract override invokeFunction(
-    invocation: Invocation,
-    abi?: Abi
+  public abstract execute(
```

Comment:
same comment as the implementation, change comment from Invoke -> call execute on account contract